jfix-stdlib-concurrency/src/test/java/ru/fix/stdlib/concurrency/futures/PendingFutureLimiterTest.java

```diff
@@ -265,6 +267,30 @@ public void limiter_pending_count_reset_on_future_with_handler() throws Exceptio
         assertEquals(0, limiter.getPendingCount());
     }
 
+    @Test
+    public void waitAll_releases_thread_after_last_future_is_completed() throws Exception {
+        PendingFutureLimiter limiter = new PendingFutureLimiter(3, TimeUnit.MINUTES.toMillis(FUTURE_LIMITER_TIMEOUT_MINUTES));
+        long taskDurationMs = 1_000;
+        Duration assertingTaskTimeout = Duration.ofMillis(taskDurationMs * 2);
+
+        limiter.enqueueBlocking(CompletableFuture.runAsync(() -> {
```

Comment:
This Runnable creation duplicated. Maybe, add smth like this:
`    private Runnable notTooLongRunningTask(long duration) {
        return () -> {
            try {
                Thread.sleep(duration);
            } catch (InterruptedException ignored) {
            }
        };
    }
`
? It should make this test more readable.

Reply:
fixed